testing/tests/DevExpress.ui.widgets/listParts/commonTests.js

```diff
@@ -1995,6 +1995,28 @@ QUnit.module('events', moduleSetup, () => {
         assert.strictEqual(actionData.itemIndex.group, 1, 'correct groupIndex passed');
     });
 
+    QUnit.test('onClick for item should be fired on \'enter\' key pressed', function(assert) {
+        if(!isDeviceDesktop(assert)) {
+            return;
+        }
+
+        const $element = this.element.dxList({
+            items:[{
+                text: 'item 1',
+                onClick: () => {
+                    assert.ok('onClick was fired');
+                }
+              }
```

Comment:
looks like missformatting
```suggestion
            }]
```